### DISCUSSION TITLE
Widget?

### DISCUSSION THREAD

**Discussion by GitAlexToTheGreek** [d4rken-org/sdmaid-se#691] [General]
Hi there - will you be adding widgets (similar to SDM1) any time soon?

Thanks,
Alex

**Comment by GitAlexToTheGreek** [comment]
Purchased the pro license for SE after the pro license for SDM1... some feedback would be appreciated; any plans to add a widget similar to SDM1's? 

![Screenshot_20231006_161311_Nova7](https://github.com/d4rken-org/sdmaid-se/assets/59787873/bc819bab-a4ac-4220-9237-ccec0df2b51e)

**Comment by d4rken** [comment]
At some point yes, it's currently not a huge priority though. I'm thinking about looking into a quick settings tile for SD Maid :thinking:
